Backoffice/AuthorizeStatusChange/Strategies/NodeStrategy.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\AuthorizeStatusChange\Strategies;
+
+use OpenOrchestra\Backoffice\AuthorizeStatusChange\AuthorizeStatusChangeInterface;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TreeNodesPanelStrategy;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelInterface\Model\StatusableInterface;
+use OpenOrchestra\ModelInterface\Model\StatusInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+/**
+ * Class NodeStrategy
+ */
+class NodeStrategy implements AuthorizeStatusChangeInterface
+{
+    /**
+     * @var AuthorizationCheckerInterface
+     */
+    protected $authorizationChecker;
+
+    /**
+     * @param AuthorizationCheckerInterface $authorizationChecker
+     */
+    public function __construct(AuthorizationCheckerInterface $authorizationChecker)
+    {
+        $this->authorizationChecker = $authorizationChecker;
+    }
+
+    /**
+     * @param StatusableInterface $document
+     * @param StatusInterface $toStatus
+     *
+     * @return bool
+     */
+    public function isGranted(StatusableInterface $document, StatusInterface $toStatus)
+    {
+        if ($document instanceof NodeInterface) {
+            if (!$this->authorizationChecker->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE)) {
+                return false;
+            }
+        }
+
+        return true;
```

Comment:
@nicolasThal : I think the lack of "support" method cause a problem here : because the strategy is not concerned by the context, it shouldn't be asked for a go/nogo. Because it is forced to return something (yes), it can have a negative impact on the security layer.
What do you think about it ?